src/HotChocolate/Language/src/Language.SyntaxTree/Extensions/TypeNodeExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static ITypeNode NullableType(this ITypeNode type)
 
         public static NamedTypeNode NamedType(this ITypeNode type)
         {
-            if (type.InnerType().InnerType().InnerType() is NamedTypeNode n)
+            if (type.InnerType().InnerType().InnerType().InnerType().InnerType() is NamedTypeNode n)
```

Comment:
@michaelstaib  Another argument against `[[[Int]]]`. 
It would not fit on a single line